specs/darwin/diagnostics.table

```diff
@@ -0,0 +1,25 @@
+table_name("diagnostics")
+description("Diagnostic information about applications executed.")
```

Comment:
I'm not very familiar with this information. Would people consider it more privacy exposure? It looks like it's only going to read `*core_analytics` I don't have that file, and I note that directory is full of program usage information.

Reply:
The artifact was introduced in 10.13 (High Sierra)
Should probably move conversation and additional comments about diagnostics to #5562?
But CrowdStrike published a pretty good [blog](https://www.crowdstrike.com/blog/i-know-what-you-did-last-month-a-new-artifact-of-execution-on-macos-10-13/) about the artifact